bin/api.sh

```diff
@@ -33,6 +33,8 @@ exec solana-validator \
   ${maybe_rpc_faucet_address} \
   --expected-genesis-hash "$EXPECTED_GENESIS_HASH" \
   --expected-shred-version "$EXPECTED_SHRED_VERSION" \
+  --expected-bank-hash "$EXPECTED_BANK_HASH" \
   --wait-for-supermajority "$WAIT_FOR_SUPERMAJORITY" \
   "${trusted_validators[@]}" \
   --no-untrusted-rpc \
+  --snapshot-version 1.1.0 \
```

Comment:
We need to set v1.1.0 snapshot version explicitly, so that when the mainnet-beta nodes upgrade to v1.2:
1. They'll continue to generate snapshots compatible with any 1.1 nodes that haven't upgraded yet
2. We can downgrade back to v1.1 if something bad happens